kiosk/kiosk_browser/proxy.py

```diff
@@ -1,13 +1,25 @@
 """Monitor proxy changes and automatically apply changes in Qt application.
 """
-import dbus
 import logging
 import threading
-from PyQt6.QtNetwork import QNetworkProxy
 from dataclasses import dataclass
-from dbus.mainloop.glib import DBusGMainLoop
-from gi.repository import GLib
-from proxy_utils import get_current_proxy
+
+from PyQt6.QtNetwork import QNetworkProxy
+
+# Try to import Linux-specific modules
+try:
```

Comment:
Maybe we should add an env variable to disable these features, to avoid them being accidentally disabled on a real system.